content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -152,14 +156,14 @@ the three things:
 
 1.  **deny** \
     If any Permit plugin denies a Pod, it is returned to the scheduling queue.
-    This will trigger [Unreserve](#unreserve) plugins.
+    This will trigger the [Unreserve](#unreserve) extension point.
```

Comment:
Once the `Unreserve` stanza is moved, we can rephrase here:

```
This will trigger the "unreserve" API in the [Reserve](#reserve) extension point.
```

Reply:
API can be replaced with other terms like "method" or "function".

Reply:
Done.